sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/PrivateService.java

```diff
@@ -303,7 +304,7 @@ public static String stdCheckFormatExceptionMessage(TPrivate key) throws ScdExce
      * @param scdRootAdapter SCL file in which Private should be found
      * @return map of Private and its IEDName parameter
      */
-    public static Stream<TPrivate> createMapIEDNameAndPrivate(SclRootAdapter scdRootAdapter) {
+    public static Stream<TPrivate> getPrivatesStreamByIEDName(SclRootAdapter scdRootAdapter) {
```

Comment:
Here, `COMPAS_ICDHEADER` private was already set in the method filter, and javadoc indicates that it is an icdHeader Private not just Private. 
```suggestion
    public static Stream<TPrivate> getICDHeaderPrivatesStreamByIEDName(SclRootAdapter scdRootAdapter) {
```

Reply:
Ok I renamed it and updated javadoc